Fix compatibility with Qt 5.15.

Patch copied from upstream source repository:

https://bazaar.launchpad.net/~adamreichold/qpdfview/trunk/revision/2104

--- a/sources/model.h	2017-04-19 21:01:25 +0000
+++ b/sources/model.h	2020-06-09 06:24:11 +0000
@@ -24,6 +24,7 @@
 #define DOCUMENTMODEL_H
 
 #include <QList>
+#include <QPainterPath>
 #include <QtPlugin>
 #include <QWidget>
 #include <QVector>

